package.json

```diff
@@ -287,6 +287,7 @@
     "@types/listr": "^0.13.0",
     "@types/lodash": "^3.10.1",
     "@types/minimatch": "^2.0.29",
+    "@types/mocha": "^5.2.5",
```

Comment:
Where is this used? AFAIK all of the tsconfig.json files extend the root config, which only lists jest, node, and react global types. We're using mocha types in `x-pack/test/tsconfig.json` but those are included by `x-pack/package.json`.

Reply:
When I was opened some test in `test` directory (e.g. `test/apps/visualize/sometest.js`) my IDE not detected typings for mocha, it was a problem. Also if we want migrate our tests in typescript (in future of course) we need to extend current `tsconfig.json` in a root, and change types block(just add mocha to them), this typings will taken from root `node_modules` folder

Reply:
Yeah, in that case we should close this PR and reopen it when we actually need the types in the repo. Your IDE shouldn't have any trouble finding the types since they're installed in the root of the repository anyway, regardless of the entry in the root package.json

Reply:
no, It haven't installed on the repo root, and i was added them in root `package.json`, one more workaround this issue is install `@types/mocha` with flag `--no-save` or `-g` and it would be installed in root/global without writing and changing `package.json` and `package-lock.json`

Reply:
You don't have to install in the root of the repo, just running `yarn kbn bootstrap` will install all of the node_modules for all packages in the root of the repo because of how we use yarn workspaces.